x/leverage/keeper/keeper.go

```diff
@@ -45,27 +44,10 @@ func (k Keeper) TotalUTokenSupply(ctx sdk.Context, uTokenDenom string) sdk.Coin
 	return sdk.NewCoin(uTokenDenom, sdk.ZeroInt())
 }
 
-// TotalTokenSupply returns an sdk.Coin representing the total balance of a
-// given asset type if valid. If the denom is not an accepted asset type, we
-// return a zero amount.
-func (k Keeper) TotalTokenSupply(ctx sdk.Context, tokenDenom string) sdk.Coin {
```

Comment:
This function can go for now (it was formerly commented //TODO: Get module account's balance of a given token) but the intended functionality will likely reappear when we start working with borrowing (supply/demand based interest rates), or potentially sooner if we derive token:uToken exchange rates a certain way.

Reply:
Yup -- I removed it 👍